src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -553,8 +553,7 @@ private Symbol ResolveMultipleSymbolsInAttributeTypeLookup(ArrayBuilder<Symbol>
                 var mdSymbol = symbols[secondBest.Index];
 
                 //if names match, arities match, and containing symbols match (recursively), ...
-                if (srcSymbol.ToDisplayString(SymbolDisplayFormat.QualifiedNameArityFormat) ==
```

Comment:
>ToDisplayString [](start = 30, length = 15)

Do we have more references to this API in the compiler? Consider opening an issue to follow up on them too.

Reply:
[https://github.com/dotnet/roslyn/issues/43206](https://github.com/dotnet/roslyn/issues/43206)

---
In reply to: [405648673](https://github.com/dotnet/roslyn/pull/43181#discussion_r405648673) [](ancestors = 405648673)